Update issue_review_template_fall25_m3.md to M4 checklist (#280)
camaraproject/ReleaseManagement: .github/ISSUE_TEMPLATE/issue_review_template_fall25_m4.md

- [ ] the release candidate suffix (`-rc.x`) has been removed from API versions
- [ ] API definition links {ReDoc, Swagger, OAS) includes the correct release tag in the URL
- [ ] Added/Changed/Removed sections includes any additional changes made for M4 and all the entries from M3
- _Note: create a "draft release" to see which PRs were done since the last document pre-release `rX.Y`_

### API definition .yaml file(s)
- [ ] `info.version` has been updated to latest version number (without rc.n suffix)
- [ ] `info.servers.url` has been updated to include latest version number (without rc.n suffix)
- [ ] check for links containing `/main/` or `/rX.Y/` (with `rX.Y` being the pre-release) which are pointing the repository. They must be replaced with the `/rX.Y+1/`.
- [ ] Read the Swagger UI view of the API yaml to spot formatting/spelling errors

### Test definition .feature file(s)
- [ ] 'Feature:' line: `vwip` has been changed to the latest version number
- [ ] All resource links have been updated with the correct serverURL and are using a root-relative path (e.g. `And the resource "/qos-profiles/vwip/qos-profiles/{name}"`)

### API Readiness checklist

> - [ ] first line ('Checklist for'): 'vwip' has been changed to the latest version number

Reviewer: ```suggestion
- [ ] first line ('Checklist for'): has been updated to the latest version number and the latest release number
- [ ] links to Commonalities and ICM r3.3
```

> 

Reviewer: Point 9 of API Readiness Checklist requires creating issue for "Test statements for stable APIs"

```suggestion
- [ ] for stable APIs Test result statement is mandatory - issue link should be referenced

```

- [ ] the `y` in `rX.Y` has been increased by 1 from the pre-release 

**Release notes**
- [ ] first paragraph: 'public release' is used instead of 'pre-release'
- [ ] the release candidate suffix (`-rc.x`) has been removed from API versions
- [ ] the release candidate suffix (`-rc.x`) has been removed from Commonalities and ICM versions

**API sections of CHANGELOG**
- [ ] the words 'release candidate' are not present
- [ ] the release candidate suffix (`-rc.x`) has been removed from API versions
- [ ] API definition links {ReDoc, Swagger, OAS) includes the correct release tag in the URL
- [ ] Added/Changed/Removed sections includes any additional changes made for M4 and all the entries from M3
- _Note: create a "draft release" to see which PRs were done since the last document pre-release `rX.Y`_

### API definition .yaml file(s)
- [ ] `info.version` has been updated to latest version number (without rc.n suffix)
- [ ] `info.servers.url` has been updated to include latest version number (without rc.n suffix)
- [ ] check for links containing `/main/` or `/rX.Y/` (with `rX.Y` being the pre-release) which are pointing the repository. They must be replaced with the `/rX.Y+1/`.

> - [ ] Read the Swagger UI view of the API yaml to spot formatting/spelling errors

Reviewer 2: Already addressed, maybe take the second part.

- [ ] the `y` in `rX.Y` has been increased by 1 from the pre-release 

**Release notes**
- [ ] first paragraph: 'public release' is used instead of 'pre-release'
- [ ] the release candidate suffix (`-rc.x`) has been removed from API versions
- [ ] the release candidate suffix (`-rc.x`) has been removed from Commonalities and ICM versions

**API sections of CHANGELOG**
- [ ] the words 'release candidate' are not present
- [ ] the release candidate suffix (`-rc.x`) has been removed from API versions
- [ ] API definition links {ReDoc, Swagger, OAS) includes the correct release tag in the URL
- [ ] Added/Changed/Removed sections includes any additional changes made for M4 and all the entries from M3
- _Note: create a "draft release" to see which PRs were done since the last document pre-release `rX.Y`_

### API definition .yaml file(s)
- [ ] `info.version` has been updated to latest version number (without rc.n suffix)
- [ ] `info.servers.url` has been updated to include latest version number (without rc.n suffix)
- [ ] check for links containing `/main/` or `/rX.Y/` (with `rX.Y` being the pre-release) which are pointing to the repository. They must be replaced with the `/rX.Y+1/`.

> - [ ] check that any properties with `date-time` format reference RFC 3339 in their description (per [Commonalities #497](https://github.com/camaraproject/Commonalities/pull/497)

Reviewer 2: ```suggestion
- [ ] check that any properties with `date-time` format reference RFC 3339 in their description (per https://github.com/camaraproject/Commonalities/pull/497). Suggest to delete the additional format recommendation if present.
```
